src/DotNetty.Common/IResourceLeak.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Common
+{
+    public interface IResourceLeak
+    {
+        /// <summary>
+        ///     Records the caller's current stack trace so that the <see cref="ResourceLeakDetector" /> can tell where the
+        ///     leaked
```

Comment:
You just have an extra linebreak here that breaks the comment onto two lines - it just looks odd